src/robusta/core/model/cluster_status.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+from pydantic.main import BaseModel, Optional
+
+class ClusterStatus(BaseModel):
+    created_at: Optional[datetime]
```

Comment:
I think we don't need created_at and updated_at here.
created_at is updated by the DB to now() the when the row is created
updated_at is always sent to the DB as now(), so no point having a member for it